runescape-api/src/main/java/net/runelite/rs/api/RSUserList.java

```diff
@@ -4,7 +4,7 @@
 
 public interface RSUserList<T extends RSUser>
 {
-	@Import("size0")
+	@Import("getSize")
```

Comment:
Should import `size` as a field instead of the `getSize` method, in case it gets inlined in the future.

Reply:
Getsize isn't static so it won't get inlined